include/nbl/builtin/hlsl/bda/__ref.hlsl

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+#include "nbl/builtin/hlsl/spirv_intrinsics/core.hlsl"
+
+#ifndef _NBL_BUILTIN_HLSL_BDA_REF_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BDA_REF_INCLUDED_
+
+namespace spirv
+{
+template<typename M, typename T, typename StorageClass>
+[[vk::ext_instruction(spv::OpAccessChain)]]
+vk::SpirvOpaqueType <spv::OpTypePointer,StorageClass,M> accessChain(
+    [[vk::ext_reference]] vk::SpirvOpaqueType <spv::OpTypePointer,StorageClass,T>base,
+    [[vk::ext_literal]] uint32_t index0
+);
+}
```

Comment:
get rid of it for now, DXC blocks us heavily here and we don't really know how to use it
https://github.com/microsoft/DirectXShaderCompiler/issues/6578
https://github.com/microsoft/DirectXShaderCompiler/issues/4924

Reply:
Done